confgenerator/confgenerator.go

```diff
@@ -193,8 +193,8 @@ func getUserAgent(prefix string, hostInfo *host.InfoStat) (string, error) {
 func getWorkers(hostInfo *host.InfoStat) int {
```

Comment:
I'd lean towards removing the conditions here for now, as we do not know when there will be a plan to diverge them. And if there is, it's simple to add it back then.